### PR TITLE
Removed documentation and now reference metrics-client-doc repository in Github

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

**Comment by matthayter** [comment]
Ah so 'load fro the internet' is the solution we're going with? This means we're okay with unlikely event of a build breaking due to internet / github issues, yeah?

File: src/test/java/com/arpnetworking/metrics/impl/TsdQueryLogSinkTest.java [hunk 1]
```diff
@@ -568,15 +572,21 @@ private String fileToString(final File file) throws InterruptedException {
...
         } catch (final IOException e1) {
             try {
-                // Under some IDE setups this may be executed from the workspace root (e.g. root)
-                jsonNode = JsonLoader.fromPath("doc/query-log-schema-2e.json");
+                // Download from the source repository
+                jsonNode = JsonLoader.fromURL(
+                        new URL("https://raw.githubusercontent.com/ArpNetworking/metrics-client-doc/master/schema/" + SCHEMA_FILE_NAME));
+
+                // Cache the schema file
+                Files.write(Paths.get("./target/" + SCHEMA_FILE_NAME), JacksonUtils.prettyPrint(jsonNode).getBytes(Charsets.UTF_8));
```
Comment (matthayter) [inline src/test/java/com/arpnetworking/metrics/impl/TsdQueryLogSinkTest.java]: Whoa. Does this cached copy persist a little when using travis or something? Or is this just for local builds?

Comment (vjkoskela) [inline src/test/java/com/arpnetworking/metrics/impl/TsdQueryLogSinkTest.java]: This is only for local builds. Travis does not persist the target directory across builds.

**Comment by vjkoskela** [comment]
Yes, we're going to give this a shot as it was low cost and seems to fit the semantics that what we are looking. The main concern with connectivity was for developers since it is possible they will work offline. Thus, the file is cached in the target directory (just don't run mvn clean while offline!). The other use case of continuous build through Travis is not as serious since either an outage or loss of connectivity to Github means Travis won't be able to build the source anyway.

**Pull request merged**